src/Bridge/Doctrine/Orm/Extension/FilterEagerLoadingExtension.php

```diff
@@ -143,4 +146,27 @@ private function isForceEager(string $resourceClass, array $options): bool
 
         return is_bool($forceEager) ? $forceEager : $this->forceEager;
     }
+
+    private function hasFetchEagerAssociation(EntityManager $em, ClassMetadataInfo $classMetadata, &$checked = [])
```

Comment:
Is it possible to extract this logic out for reuse between `EagerLoadingExtension` and `FilterEagerLoadingExtension`?